README.md

```diff
@@ -1 +1,193 @@
-# awesome-book
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <h3><b>Awesome Books Project</b></h3>
+</div>
+# :green_book: Table of Contents
+
+- [:book: About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [:rocket: Live Demo](#live-demo)
+- [:computer: Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [:bust_in_silhouette: Author](#author)
+- [:telescope: Future Features](#future-features)
+- [:handshake: Contributing](#contributing)
+- [:star:️ Show your support](#support)
+- [:pray: Acknowledgements](#acknowledgements)
+- [:memo: License](#license)
+
+# :book: Awesome Books Project <a name="Awesome Books Project"></a>
+
+**Awesome Books Project** is the  project of module 2.It is a simple website and it can be used for saving books. Here we used HTML, CSS to design site and used Javascript for dynamic functionality.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">HTML</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Client-side_web_APIs/Manipulating_documents">DOM</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object">JavaScript Object</a></li>
+    <li><a href="https://css-tricks.com/snippets/css/a-guide-to-flexbox/">FlexBox</a></li>
+    <li><a href="https://mozilla.github.io/addons-linter/">Linters</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **Using JavaScript Object**
+- **Local Storage**
+- **Dynamics of HTML using JavaScript**
+- **DOM Manipulation**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link]()
```

Comment:
- [ ] Live demo link is broken. Kindly update the link or delete it if the project does not require deployment 👍🏾